examples/config-rust/.env

```diff
@@ -1 +0,0 @@
-SPIN_CONFIG_DOTENV = "dotenv"
```

Comment:
Is this file still there but empty? Is that what we want?

Reply:
`config-rust` was renamed to `variables-rust` so this file exists there and the change is to use `SPIN_VARIABLE_DOTENV`.